data/diagnostics.json

```diff
@@ -0,0 +1,250 @@
+{
+  "bitLimits": [
```

Comment:
`bitLimits` currently based on the package viewer's error validation - it should be extracted from the protobuff files in the python codebase instead